src/grid-list/grid-list.jsx

```diff
@@ -15,6 +15,7 @@ const GridList = React.createClass({
     cols: React.PropTypes.number,
     padding: React.PropTypes.number,
     cellHeight: React.PropTypes.number,
+    cellWidth: React.PropTypes.number,
```

Comment:
I deliberately didn't add a default value so that the existing 'cols'
property default would make that the default operation.  That way it
wouldn't break existing defaults when someone upgrades to latest version of
material-ui.

On Thursday, 26 November 2015, Olivier Tassinari <notifications@github.com>
wrote:

> In src/grid-list/grid-list.jsx
> <https://github.com/callemall/material-ui/pull/2274#discussion_r46008161>:
>
> > @@ -15,6 +15,7 @@ const GridList = React.createClass({
> >      cols: React.PropTypes.number,
> >      padding: React.PropTypes.number,
> >      cellHeight: React.PropTypes.number,
> > +    cellWidth: React.PropTypes.number,
>
> What do you think about adding a default value of 'auto'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/callemall/material-ui/pull/2274/files#r46008161>.
>